vsts/pipelines/weeklyCleaning.yml

```diff
@@ -0,0 +1,61 @@
+variables:
```

Comment:
nit: Rename the file to like `agentCleanup.yml`? We might want to change the trigger timeline to biweekly in which case this file name is incorrect.